mosquitto/README.md

```diff
@@ -0,0 +1,101 @@
+
+## Installation
+
+The installation of this add-on is straightforward and easy to do.
+
+1. Navigate in your Home Assistant frontend to **Hass.io** -> **Add-on Store**.
+2. Find the "Mosquitto broker" add-on and click it.
+3. Click on the "INSTALL" button.
+
+## How to use
+
+The add-on has a couple of options available. For more detailed instructions see [mosquitto]. The basic thing to get the add-on running would be:
+
+1. Start the add-on.
+2. Have some patience and wait a couple of minutes.
+3. Check the add-on log output to see the result.
+
+## Configuration
+
+Add-on configuration:
+
+```json
+{
+  "logins": [],
+  "anonymous": false,
+  "customize": {
+    "active": false,
+    "folder": "mosquitto"
+  },
+  "certfile": "fullchain.pem",
+  "keyfile": "privkey.pem",
+  "require_certificate": false
+}
+```
+
+### Option: `logins`
+
+A list of local users that will be created with username and password. You don’t need do this because you can use Home Assistant users too without any configuration.
+
+### Option: `anonymous`
+
+Allow anonymous connections. If logins is set, the anonymous user can only read data.
+
+### Option: `customize`
+
+This option allows you to provide additional configuration files as needed.
+
+#### Option: `customize` -> `active`
+
+If set to true additional configuration files will be read, see next option.
+
+#### Option: `customize` -> `folder`
+
+The folder to read the  additional configuration files (*.conf) from.
+
+### Option: `certfile`
+
+File containing certificate including its chain.
+
+### Option: `keyfile`
+
+File containing the private key.
+
+### Option: `require_certificate`
+
+If set to true encryption will be enabled using the cert- and keyfile options.
+
+## Known issues and limitations
+
+## Support
+
+Got questions?
+
+You have several options to get them answered:
+
+- The [Home Assistant Discord Chat Server][discord].
+- The Home Assistant [Community Forum][forum].
+- Join the [Reddit subreddit][reddit] in [/r/homeassistant][reddit]
+
+In case you've found an bug, please [open an issue on our GitHub][issue].
+
+[aarch64-shield]: https://img.shields.io/badge/aarch64-yes-green.svg
+[amd64-shield]: https://img.shields.io/badge/amd64-yes-green.svg
+[armhf-shield]: https://img.shields.io/badge/armhf-yes-green.svg
+[armv7-shield]: https://img.shields.io/badge/armv7-yes-green.svg
+[discord]: https://discord.gg/c5DvZ4e
+[forum]: https://community.home-assistant.io
+[i386-shield]: https://img.shields.io/badge/i386-yes-green.svg
+[issue]: https://github.com/home-assistant/hassio-addons/issues
+[reddit]: https://reddit.com/r/homeassistant
+[repository]: https://github.com/hassio-addons/repository
+[mosquitto]: https://www.home-assistant.io/addons/mosquitto/
```

Comment:
This README should be a replacement for the URL you are referring to. Either we should pull in information from that URL into this document or point elsewhere.

Reply:
You mean deprecation of this (https://www.home-assistant.io/addons/mosquitto/) entire page and having all the content moved over?

Reply:
Our goal is to have add-on provide the documentation right from the HA interface (add-on store and add-on config page). That removes the need to keep our website updated.
So our current goal is getting good documentation on the add-ons, so we can start proceeding on removing/redirecting users from the website to these documents.

It is easier to keep these into sync with the add-on as well.

Reply:
Done, see https://github.com/home-assistant/hassio-addons/pull/748/commits/be1264216a48e5a4c3afdf0e60ff5dbcca0af5d5